group-generators/helpers/data-providers/ens-subdomain/index.ts

```diff
@@ -0,0 +1,94 @@
+import { gql } from "graphql-request";
+import { EnsSubdomainsResponse, EnsDomainParams } from "./types";
+import { GraphQLProvider } from "@group-generators/helpers/data-providers/graphql";
+
+import { FetchedData } from "topics/group";
+
+export class EnsSubdomainProvider extends GraphQLProvider {
+  constructor() {
+    super({
+      url: `https://gateway.thegraph.com/api/${EnsSubdomainProvider.getAPIKey()}/subgraphs/id/EjtE3sBkYYAwr45BASiFp8cSZEvd1VHTzzYFvJwQUuJx`,
+    });
+  }
+
+  public async getEnsSubdomains({
+    subdomain,
```

Comment:
I think the argument name here is more "domain" than "subdomain"
(same for all other "subdomain" iterations)

Reply:
Thanks! That's more logical